mmv1/products/certificatemanager/DnsAuthorization.yaml

```diff
@@ -84,6 +91,22 @@ properties:
       A domain which is being authorized. A DnsAuthorization resource covers a
       single domain and its wildcard, e.g. authorization for "example.com" can
       be used to issue certificates for "example.com" and "*.example.com".
+  - !ruby/object:Api::Type::Enum
+    name: type
+    description: |
+      type of DNS authorization. If unset during the resource creation, FIXED_RECORD will
+      be used for global resources, and PER_PROJECT_RECORD will be used for other locations.
+
+      FIXED_RECORD DNS authorization uses DNS-01 validation method
+
+      PER_PROJECT_RECORD DNS authorization allows for independent management
+      of Google-managed certificates with DNS authorization across multiple
+      projects.
+    immutable: true
+    values:
+    - :FIXED_RECORD
```

Comment:
Getting a lint error due to indentation of each value in `values`:
` Error: 107:5 [indentation] wrong indentation: expected 6 but found 4`

Reply:
Done.